src/singpass-button/types.ts

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { SingpassLogoOverride } from "./singpass-button.style";
+
+export type SingpassButtonStyleType = "red-filled" | "white-filled";
+
+export interface SingpassButtonProps
+    extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+    "data-testid"?: string | undefined;
+    /** The style type of the button. Values: "red-filled" | "white-filled" */
+    styleType?: SingpassButtonStyleType | undefined;
+    singpassLogoStyleOverride?: SingpassLogoOverride | undefined;
```

Comment:
propose to drop this override prop

if it does need to be overridden because it doesn't work with your font, it probably should be done on the theme level

Reply:
removed since we're now using svg